tiledb/sm/array_schema/dimension.cc

```diff
@@ -842,10 +831,8 @@ void Dimension::relevant_ranges<char>(
     const auto& r1_start = ranges[r].start_str();
     const auto& r1_end = ranges[r].end_str();
 
-    const auto r1_after_r2 =
-        !r1_start.empty() && !mbr_end.empty() && r1_start > mbr_end;
-    const auto mbr_after_r1 =
-        !mbr_start.empty() && !r1_end.empty() && mbr_start > r1_end;
+    const auto r1_after_r2 = r1_start > mbr_end;
+    const auto mbr_after_r1 = mbr_start > r1_end;
```

Comment:
(seems prior to your changes)
curious naming, r1, r2, when there is essentially only 'r', and no r2 anywhere in routine I can see...

Reply:
Good catch, removed.